java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -158,6 +158,14 @@ public static void assertPartialColumnsAreEqual(HostColumnVector expected, long
             assertArrayEquals(expected.getUTF8(expectedRow), cv.getUTF8(tableRow),
                 "Column " + colName + " Row " + tableRow);
             break;
+          case LIST:
+            try {
+              assertArrayEquals(expected.getList(expectedRow).toString().getBytes(), cv.getList(tableRow).toString().getBytes(),
```

Comment:
Can we have a follow on issue to fix this please?  We should not be doing string comparison for the lists.  This makes it so we cannot do approximate float comparisons or make sure that the types match or null count matches for a list of strings that has nulls in them....

Reply:
Yes. agree this is not grt.